lib/Driver.js

```diff
@@ -3,7 +3,7 @@
  */
 'use strict';
 
-const {LDKeyPair} = require('crypto-ld');
+const {Ed25519KeyPair, LDKeyPair} = require('crypto-ld');
```

Comment:
Is `Ed25519KeyPair` "built-in" to crypto-ld? I'd think there's another package that needs to be brought in for it.

```suggestion
const {Ed25519KeyPair, LDKeyPair} = require('crypto-ld');
```

Reply:
It's built in for the version of crypto-ld that did-method-key-js is using. I was attempting to avoid a major refactor, but that looks like what you're asking for here.

Reply:
https://github.com/digitalbazaar/ed25519-verification-key-2018